Docker: No module named 'flask_restful'

I am failing to run this using Docker as described [in the docs](https://ihatemoney.readthedocs.io/en/latest/installation.html#with-docker). Any help?

##### Steps to reproduce:

    $ git clone https://github.com/spiral-project/ihatemoney.git && cd ihatemoney
    $ docker build -t ihatemoney --build-arg INSTALL_FROM_PYPI=True .
    $ docker run -d -p 8000:8000 ihatemoney
    $ docker ps

##### Expected behavior:

running Docker container

##### Actual behavior:

exited Docker container

    $ docker logs dead_ihatemoney_container
    [2018-01-30 16:51:08 +0000] [7] [INFO] Starting gunicorn 19.7.1
    [2018-01-30 16:51:08 +0000] [7] [INFO] Listening at: http://0.0.0.0:8000 (7)
    [2018-01-30 16:51:08 +0000] [7] [INFO] Using worker: sync
    [2018-01-30 16:51:08 +0000] [10] [INFO] Booting worker with pid: 10
    [2018-01-30 16:51:08 +0000] [11] [INFO] Booting worker with pid: 11
    [2018-01-30 16:51:08 +0000] [12] [INFO] Booting worker with pid: 12
    [2018-01-30 16:51:10 +0000] [11] [ERROR] Exception in worker process
    Traceback (most recent call last):
      File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 578, in spawn_worker
        worker.init_process()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 126, in init_process
        self.load_wsgi()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 135, in load_wsgi
        self.wsgi = self.app.wsgi()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
        self.callable = self.load()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
        return self.load_wsgiapp()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
        return util.import_app(self.app_uri)
      File "/usr/local/lib/python3.6/site-packages/gunicorn/util.py", line 352, in import_app
        __import__(module)
      File "/ihatemoney/ihatemoney/wsgi.py", line 1, in <module>
        from ihatemoney.run import create_app
      File "/ihatemoney/ihatemoney/run.py", line 12, in <module>
        from ihatemoney.api import api
      File "/ihatemoney/ihatemoney/api.py", line 3, in <module>
        from flask_restful import Resource, Api, abort
    ModuleNotFoundError: No module named 'flask_restful'
    [2018-01-30 16:51:10 +0000] [12] [ERROR] Exception in worker process
    Traceback (most recent call last):
      File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 578, in spawn_worker
        worker.init_process()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 126, in init_process
        self.load_wsgi()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 135, in load_wsgi
        self.wsgi = self.app.wsgi()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
        self.callable = self.load()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
        return self.load_wsgiapp()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
        return util.import_app(self.app_uri)
      File "/usr/local/lib/python3.6/site-packages/gunicorn/util.py", line 352, in import_app
        __import__(module)
      File "/ihatemoney/ihatemoney/wsgi.py", line 1, in <module>
        from ihatemoney.run import create_app
      File "/ihatemoney/ihatemoney/run.py", line 12, in <module>
        from ihatemoney.api import api
      File "/ihatemoney/ihatemoney/api.py", line 3, in <module>
        from flask_restful import Resource, Api, abort
    ModuleNotFoundError: No module named 'flask_restful'
    [2018-01-30 16:51:10 +0000] [11] [INFO] Worker exiting (pid: 11)
    [2018-01-30 16:51:10 +0000] [10] [ERROR] Exception in worker process
    Traceback (most recent call last):
      File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 578, in spawn_worker
        worker.init_process()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 126, in init_process
        self.load_wsgi()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 135, in load_wsgi
        self.wsgi = self.app.wsgi()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
        self.callable = self.load()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
        return self.load_wsgiapp()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
        return util.import_app(self.app_uri)
      File "/usr/local/lib/python3.6/site-packages/gunicorn/util.py", line 352, in import_app
        __import__(module)
      File "/ihatemoney/ihatemoney/wsgi.py", line 1, in <module>
        from ihatemoney.run import create_app
      File "/ihatemoney/ihatemoney/run.py", line 12, in <module>
        from ihatemoney.api import api
      File "/ihatemoney/ihatemoney/api.py", line 3, in <module>
        from flask_restful import Resource, Api, abort
    ModuleNotFoundError: No module named 'flask_restful'
    [2018-01-30 16:51:10 +0000] [12] [INFO] Worker exiting (pid: 12)
    [2018-01-30 16:51:10 +0000] [10] [INFO] Worker exiting (pid: 10)
    Traceback (most recent call last):
      File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 209, in run
        self.sleep()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 359, in sleep
        ready = select.select([self.PIPE[0]], [], [], 1.0)
      File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 244, in handle_chld
        self.reap_workers()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 524, in reap_workers
        raise HaltServer(reason, self.WORKER_BOOT_ERROR)
    gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>

    During handling of the above exception, another exception occurred:

    Traceback (most recent call last):
      File "/usr/local/bin/gunicorn", line 11, in <module>
        sys.exit(run())
      File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 74, in run
        WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/app/base.py", line 203, in run
        super(Application, self).run()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/app/base.py", line 72, in run
        Arbiter(self).run()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 231, in run
        self.halt(reason=inst.reason, exit_status=inst.exit_status)
      File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 344, in halt
        self.stop()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 393, in stop
        time.sleep(0.1)
      File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 244, in handle_chld
        self.reap_workers()
      File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 524, in reap_workers
        raise HaltServer(reason, self.WORKER_BOOT_ERROR)
    gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>

--- a/Dockerfile
+++ b/Dockerfile
@@ -4,12 +4,11 @@ RUN mkdir /ihatemoney &&\
     mkdir -p /etc/ihatemoney &&\
     pip install --no-cache-dir gunicorn pymysql
 
-WORKDIR /ihatemoney
-COPY . .
+COPY . /ihatemoney
 ARG INSTALL_FROM_PYPI="False"
 RUN if [ "$INSTALL_FROM_PYPI" = True ]; then\
     pip install --no-cache-dir ihatemoney ; else\
-    pip install --no-cache-dir -e . ; \
+    pip install --no-cache-dir -e /ihatemoney ; \
     fi
 
 ENV DEBUG="False" \

